### PR TITLE
[Testing] Test coverage for Home

### PR DESCRIPTION
**Pull request by sarahscott**
- [x] Basic snapshot
- [x] Removed regex
- [x] Finish logic tests

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>No CHANGELOG added.</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by mennenia** [comment]
@sarahscott could you add changelog please? ^_^

**Comment by sarahscott** [comment]
done! 💫

**Comment by orta** [comment]
Looks like Flow is failing on your build too ^

File: lib/containers/__tests__/home-tests.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  it('renders as expected', () => {
+    const homeComponent = renderer.create(<Home home={homeProps(3,3,3).home}/>).toJSON()
+    expect(homeComponent).toMatchSnapshot()
+  })
+})
+
+var homeProps = (nHeroUnits?: number, nArtistModules?: number, nArtworkModules?: number) => {
+  return {
+      home: {
+        hero_units: nHeroUnits ? modulesOfType('hero_units', nHeroUnits) : [],
```
Comment (sarahscott) [inline lib/containers/__tests__/home-tests.js]: This should fix the flow errors except the `import { ListViewDataSource }` one, which could be related to [this](https://github.com/flowtype/flow-typed/issues/16) and [this](https://github.com/facebook/flow/issues/1425)

**Comment by mennenia** [comment]
Looks good @sarahscott! I know you had a question for Orta about that import, so I'll leave this up for now. Otherwise, 🎉 💃 awesome stuff.

File: lib/containers/__tests__/home-tests.js (hunk 1 above)
Comment (orta) [inline lib/containers/__tests__/home-tests.js]: As we know this is a flow issue, it's best to use the flow ignore error syntax to skip the error:

```js
// $FlowFixMe: This is a bug with Flow + React-Native
import { ListViewDataSource } 
```

Comment (sarahscott) [inline lib/containers/__tests__/home-tests.js]: It turns out I wasn't even using `ListViewDataSource` anymore 😁  so I removed that line, but FWIW `// $FlowFixMe` worked like a charm for suppressing the error.

**Comment by sarahscott** [comment]
🍏 📗 💚 🥗 🇬🇱

**Pull request merged**